cogs/embeds.py

```diff
@@ -0,0 +1,425 @@
+import json
+import textwrap
+import typing as t
+from collections import defaultdict
+from ast import literal_eval
+
+from discord import Color, Embed, Member, TextChannel
+from discord.errors import HTTPException
+from discord.ext.commands import Bot, Cog, ColourConverter, Context, group, Converter
+
+
+# TODO; Move this into converters file
+class Unicode(Converter):
+    """Convert raw input into unicode formatted string"""
+
+    def process_unicode(self, message: str) -> str:
+        """
+        This accepts any string with raw unicode and converts it into proper unicode.
+
+        It uses literal eval to process the string safely and turn it into proper unicode.
+        """
+
+        # Only process individual lines to avoid EOL in expression
+        lines = message.split("\n")
+        for index, line in enumerate(lines):
+            try:
+                # Replace ''' which would exit the string
+                # even though it won't be allowed by literal_eval, it is still better
+                # to replace it as it will properly evaluate even strings with that.
+                line = line.replace("'''", "`<ESCAPE STRING>`")
+                line = literal_eval(f"'''{line}'''")
+                line = line.replace("`<ESCAPE STRING>`", "'''")
+                lines[index] = line
+            except SyntaxError as e:
+                print(line)
+                print(f"String deemed unsafe -> {e}")
+
+        return "\n".join(lines)
+
+    def outside_delimeter(self, string: str, delimeter: str, operation: t.Callable) -> str:
+        """Apply given operation to text outside of delimeted section"""
+        splitted = string.split(delimeter)
+        for index, string_part in enumerate(splitted):
+            # Not inside of a delimeted section
+            if index % 2 == 0:
+                splitted[index] = operation(string_part)
+
+        return delimeter.join(splitted)
+
+    async def convert(self, ctx: Context, message: str) -> str:
+        # TODO: don't replace unicode characters within code blocks
+        operation = lambda x: self.outside_delimeter(x, "`", self.process_unicode)
+        return self.outside_delimeter(message, "```", operation)
+
+
+class EmbedData(t.NamedTuple):
+    """Data for user embeds."""
+
```

Comment:
```suggestion
```

Reply:
Actually it's quite common to have space under a class docstring, this doesn't apply to function docstrings, but with classes it's acceptable and I believe black would even enforce this if we'd still be using it.